web-ui/src/styles/_variables.css

```diff
@@ -0,0 +1,100 @@
+:root {
+  /** Breakpoints
+      (more of a reference unless we bring in 
+      postcss & custom-media)
+  */
+  --sm: 600px;
+  --md: 900px;
+  --lg: 1200px;
+  --xl: 1536px;
+
+  /* Colors */
+  --oci-light-blue: #76c8d4;
+  --oci-dark-blue: #2c519e;
+  --oci-orange: #f8b576;
+  --black: #121212;
+  --white: #fff;
+  --gray-dark: #555353;
+  --gray-light: #e5e5e5;
+  --gray: #808080;
+  --red: #f00;
+  --yellow: #ee8c00;
+  --green: #006400;
+
+  --primary: var(--oci-dark-blue);
+  --primary-light: color-mix(in oklab, var(--oci-dark-blue), var(--white) 10%);
+  --primary-dark: #1a2e4f;
+  --secondary: var(--oci-orange) --secondary-main: var(--oci-light-blue);
+  --primary-text: var(--black);
+  --primary-link-light: var(--primary-light);
+  --primary-link-light-hover: #6084d1;
+  --primary-link-dark: color-mix(in oklab, var(--oci-dark-blue), var(--white) 35%);
+  --primary-link-dark-hover: #4f5f8c;
+  --action: color-mix(in oklab, var(--black), var(--white) 50%);
+  --action-disabled: color-mix(in oklab, var(--black), var(--white) 5%);
+
+  --checkins-palette-action-disabledOpacity: 0.58;
+
+  /* MUI Prefixes */
```

Comment:
What is the connection between these variables and MUI? Are they only used to style MUI components?